lib/unit.fz

```diff
@@ -92,3 +92,8 @@ unit is
 
   # human readable string
   redef asString => "unit"
+
+  type.monoid : Monoid unit is
```

Comment:
please add an explanation what this does and what it is good for. Even though this is inherently simple, it is not at all obvious for a newbie...